runtime/doc/nvim-extra-keymaps.txt

```diff
@@ -0,0 +1,36 @@
+*nvim-extra-keymaps.txt*    For nvim.                         {Nvim}
+						    Last change:      2015 May
+
+EXTRA KEYMAPS FOR NVIM                                      *nvim-extra-keymaps*
```

Comment:
Sections seem to be defined this way, and there is a syntax group for this (`helpHeadline`). It seems like the other nvim_ document files not using it should be modified... (I'll follow any guideline you guys come up with, though).